pkg/plugins/plugindef/plugindef.cue

```diff
@@ -125,7 +125,7 @@ schemas: [{
 
 			// Required Grafana version for this plugin. Validated using
 			// https://github.com/npm/node-semver.
-			grafanaDependency: =~"^(<=|>=|<|>|=|~|\\^)?([0-9]+)(\\.[0-9x\\*]+)(\\.[0-9x\\*]+)?(\\s(<=|>=|<|=>)?([0-9]+)(\\.[0-9x]+)(\\.[0-9x]+))?$"
+			grafanaDependency?: =~"^(<=|>=|<|>|=|~|\\^)?([0-9]+)(\\.[0-9x\\*]+)(\\.[0-9x\\*]+)?(\\s(<=|>=|<|=>)?([0-9]+)(\\.[0-9x]+)(\\.[0-9x]+))?$"
```

Comment:
i had to do this so that core plugins' `plugin.json` would pass basic validation, which is required to being able to load their composable kinds, and we need to load them to compose them!

It seems like this change is a few months old, which is baffling to me - it should have blown up `make gen-cue` from the moment the change was made. I recognize we probably don't want to actually relax this constraint, but i'd love it if we could do this for now just to get things moving, then we figure out what's going on there in a follow-up.